server/services/receive.js

```diff
@@ -4,10 +4,13 @@ const path = require('path');
 
 const log = bunyan.createLogger({ name: 'receive-service' });
 
-const Response = require(path.join(__dirname, './response'));
+const { RescueModel } = require(path.join(__dirname, '../models/'));
 const GraphAPi = require(path.join(__dirname, './graph-api'));
 const Rescue = require(path.join(__dirname, './rescue'));
+const Response = require(path.join(__dirname, './response'));
 const Update = require(path.join(__dirname, './update'));
+const { isValidRescue } = require('../utils/helpers');
+const { genText } = require('./response');
```

Comment:
Better to use the path module as it handles file paths regardless of the environment